api/python/t4/packages.py

```diff
@@ -116,6 +117,13 @@ def _clone(self):
 class Package(object):
     """ In-memory representation of a package """
 
+    @staticmethod
+    def validate_package_name(name):
+        """ Verify that a package name is two alphanumerics strings separated by a slash."""
+        if not re.match(r"[a-zA-Z0-9_\\-]+/[a-zA-Z0-9_\\-]+$", name):
```

Comment:
Either `r` prefix or double backslash; this one allows backslashes in the package name.
Also, make the regex a global const in `util.py` or something? Could pre-compile it, too.